asm/non_matchings/code_80005FD0/func_80009B60.s

```diff
@@ -171,7 +171,7 @@ glabel func_80009B60
 /* 00A988 80009D88 1000001A */  b     .L80009DF4
 /* 00A98C 80009D8C 8FA50040 */   lw    $a1, 0x40($sp)
 .L80009D90:
-/* 00A990 80009D90 0C004B70 */  jal   func_80012DC0
+/* 00A990 80009D90 0C004B70 */  jal   train_collision
```

Comment:
Unfortunately, disabling this function the trains still have collision.

I have no idea what this function does.

Reply:
I revert it then